internal/aghtls/aghtls.go

```diff
@@ -28,3 +33,14 @@ func SaferCipherSuites() (safe []uint16) {
 
 	return safe
 }
+
+func UserPreferredCipherSuites(ciphers []string) (userCiphers []uint16) {
+	for _, s := range tls.CipherSuites() {
+		if slices.Contains(ciphers, s.Name) {
+			userCiphers = append(userCiphers, s.ID)
+			log.Debug("user specified cipher : %s, ID : %d", s.Name, s.ID)
+		}
+	}
```

Comment:
Currently, this function ignores unknown ciphers, which seems like a mistake.  It should return an informative error instead.

Reply:
logged unknown ciphers

Reply:
Logging is not strict enough.  Please return an error and process it making AGH quit if the user has entered an incorrect cipher ID.